independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/core/strategy/SmartConfigMergeCodestartFileStrategyHandler.java

```diff
@@ -1,5 +1,7 @@
 package io.quarkus.devtools.codestarts.core.strategy;
 
+import static io.quarkus.devtools.codestarts.quarkus.QuarkusCodestartData.QuarkusDataKey.APP_CONFIG;
```

Comment:
@cristian-com here you can't use this as this create a circular dependency. At some point I will move those specific `strategies` to `devtools-common` but it's not done yet. Until then just hard code the key in a constant here for now.